src/canonicalBookMap/README.md

```diff
@@ -0,0 +1,16 @@
+Steps to create a map of book modules:
+1. Copy the data from mapping spreadsheet. Expected column order: 2e Module, 2e UUID, 3e Module, 3e UUID
+
+2. Paste the copied data into template literal
+    const data = `PASTED DATA`
+
+3. Convert the string into array
+    const mappingList = data.split('\n').map((line) => line.split('\t'))
+
+4. Create the map of modules as a string (Modify the moduleMapComment if necessay)
+    const modulesMapStr = mappingList.reduce((prev, current) => {
+        const moduleMapComment = current[0] === current[2] ? `${current[0]} to the same module in 3e` : `${current[0]} to the ${current[2]}`
```

Comment:
According to our conversation with Tom: https://app.zenhub.com/workspaces/openstax-unified-5b71aabe3815ff014b102258/issues/openstax/unified/1595#issuecomment-848812612
it seems like these names not always will be accurate so the single name of the module should be enough